python/anvill/binja.py

```diff
@@ -238,16 +238,52 @@ def _convert_bn_type(tinfo: bn.types.Type, cache):
         width = tinfo.width
         return FloatingPointType(width)
 
+    elif tinfo.type_class == bn.TypeClass.NamedTypeReferenceClass:
+        named_tinfo = tinfo.named_type_reference
+        if (named_tinfo.type_class
+            == bn.NamedTypeReferenceClass.StructNamedTypeClass):
+            # Add array of size width as struct element type
+            # TODO(akshayk) How to recover the struct elements type??
```

Comment:
This doesn't seem right; what's going on here? I think you want to call `add_element_type` multiple  times.

Reply:
I could not find Binary ninja API's to get element members of struct Named type. So the elements are getting recovered as an array of size width.

I created an array [i8 x64] and added it as an element of the struct. The type will be recovered as ` "type": "=0{[Bx64]}"` and lifted as `%anvill.struct.0 = type { [64 x i8] }` 

The other solution could be adding the bytes type multiple time by calling the function `add_element_type`.